pilot/pkg/security/authz/model/model.go

```diff
@@ -193,6 +194,7 @@ func generatePermission(rl ruleList, forTCP, forDeny bool) (*rbacpb.Permission,
 	for _, r := range rl.rules {
 		ret, err := r.permission(forTCP, forDeny)
 		if err != nil {
+			fmt.Errorf("permission istio/pilot/pkg/security/authz/model/model.go line197")
```

Comment:
I suggest not specify the code line number. As the code are always changing. You can print out something like:

fmt.Errorf("failed to generate permission, error is %v", err)